packages/fishjam-openapi/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@fishjam-cloud/fishjam-openapi",
   "version": "0.3.0",
+  "private": true,
```

Comment:
you should also update `release.yml` to stop publishing these packages (and probably mark them as deprecated?)